Jenkinsfile

```diff
@@ -5,71 +5,84 @@ properties([
 ])
 
 parallel(
-    cloud: {
-        node('cloud') {
-            deleteDir()
-            checkout scm
-            try {
-                parallel(
-                    x86_64_test: {
-                        stage('X86_64: Run the test suite') {
-                            sh 'virtualenv -p python3 venv'
-                            sh './venv/bin/python ./venv/bin/pip install -r requirements.txt'
-                            sh './venv/bin/python ./venv/bin/pytest'
-                        }
-                    },
-                    x86_64_build: {
-                        stage('X86_64: Build a reusable image') {
-                            sh './factory prepare-cloud-image'
-                        }
-                        stage('X86_64: Check the image') {
-                            sh './factory run true'
-                        }
-                        stage('X86_64: Save artifacts') {
-                            sh 'cd images/cloud-x86_64 && tar c * > ../../cloud-x86_64-image.tar'
-                            sh 'xz -0 < cloud-x86_64-image.tar > cloud-x86_64-image.tar.xz'
-                            archiveArtifacts 'cloud-x86_64-image.tar'
-                            archiveArtifacts 'cloud-x86_64-image.tar.xz'
-                        }
-                    }
-                )
-            } finally {
-                deleteDir()
-            }
+
+  x86_64_test: {
+    node('cloud') {
+      deleteDir()
+      checkout scm
+      try {
+        stage('X86_64: Run the test suite') {
+          sh 'virtualenv -p python3 venv'
+          sh './venv/bin/python ./venv/bin/pip install -r requirements.txt'
+          sh './venv/bin/python ./venv/bin/pytest'
```

Comment:
add `--junitxml x86_64_test/results.xml` here, and collect with `junit x86_64_test/results.xml`